packages/lesswrong/components/search/PostsListEditorSearchHit.tsx

```diff
@@ -46,21 +52,23 @@ const PostsListEditorSearchHit = ({hit, classes}: {
       >
         <PostsPreviewTooltipSingle postId={post._id} postsList/>
       </LWPopper>
-      <div>
+      <div className={classes.titleRow}>
         <PostsTitle post={post as unknown as PostsListBase} isLink={false}/>
       </div>
-      {post.authorDisplayName && <MetaInfo>
-        {post.authorDisplayName}
-      </MetaInfo>}
-      <MetaInfo>
-        {post.baseScore} points
-      </MetaInfo>
-      {post.postedAt && <MetaInfo>
-        <FormatDate date={post.postedAt}/>
-      </MetaInfo>}
-      <Link to={postGetPageUrl(post)} className={classes.postLink}>
-        (Link)
-      </Link>
+      <div className={classes.metadataRow}>
```

Comment:
[nit] Unnecessary div

Reply:
And empty class above

Reply:
I think this isn't actually unnecessary, it changes the wrapping by being a block. The class is unnecessary since it has no stylign though.